director/projects/api/urls.py

```diff
@@ -16,8 +18,9 @@
     ProjectEventListView,
     AdminProjectEventListView,
 )
+from projects.api.views.nodes import NodesViewSet
 
-urlpatterns = [
+projects_urls = [
```

Comment:
Is there a reason for this change?

Reply:
I did that because there is now a `nodes_urls` in that file. I was thinking that we might have other url groups within the `projects` app that would use the same naming convention (e.g. `files_urls`. But I've since changed my on that and think that nodes are a special case (which are not nested under a `projects/<project_id>/...` i.e. you don't shouldn't nee the project id to retrieve the node). I'm happy to change back to `urlpatterns` if you think that is better.

Reply:
Django convention is `urlpatterns` and then you can just 'include` the dotted path to the file and it will automatically load that, multiple `urlpatterns` require multiple files. One one hand it would be good to stick to convention but there's nothing stopping you just importing the variable you need manually and using that. So whichever you think will lead to less code in the end is probably the way to go.

Reply:
OK, will leave as is for the moment.